pkg/manager/task_scan.go

```diff
@@ -1208,14 +1221,15 @@ func (t *ScanTask) rescanImage(i *models.Image, fileModTime time.Time) (*models.
 	return ret, nil
 }
 
-func (t *ScanTask) associateImageWithFolderGallery(imageID int, qb models.GalleryReaderWriter) error {
+func (t *ScanTask) associateImageWithFolderGallery(imageID int, qb models.GalleryReaderWriter) (int, bool, error) {
```

Comment:
Just for readability issues, should the function be adjusted so that the return values are named (eg `galleryID int, isNew bool`)?

Reply:
Good suggestion, updated 👍